cloud_functions/climateiq_data_export_cf/requirements.txt

```diff
@@ -1,64 +1,14 @@
-affine==2.4.0
```

Comment:
Did you mean to remove all these requirements?
The requirements file should contain the transitive closure over all requirements.

Reply:
Got it, reverted.